angular/app.module.js

```diff
@@ -0,0 +1,8 @@
+var AppModule = ng.core.NgModule({
+    imports: [ng.platformBrowser.BrowserModule, ng.forms.FormsModule],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ng' is not defined.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5238370)